app/src/backend/dependency.ts

```diff
@@ -18,6 +18,11 @@ let platform = os.platform();
 if (!platform)
   throw new Error(`Unrecognized platform. Must be Windows, Mac, or Ubuntu.`);
 
+let startInitSJCloud: any;
```

Comment:
@jsunny23 I don't like that these variable are kept at the global scope. Move these to function-level.